packages/venia-ui/lib/components/CheckoutPage/checkoutPage.js

```diff
@@ -57,25 +57,28 @@ const CheckoutPage = props => {
         setPaymentInformationDone,
         resetReviewOrderButtonClicked,
         handleReviewOrder,
-        reviewOrderButtonClicked
+        reviewOrderButtonClicked,
+        revertPaymentInformationDone
     } = talonProps;
 
     const [, { addToast }] = useToasts();
 
     useEffect(() => {
         if (hasError) {
+            const message =
+                error && error.message
+                    ? error.message
```

Comment:
Can we display the same error we receive from the GraphQl server? At times, it can contain text like `GQL Error: *******` which might not be useful for the user.